pylint/lint/pylinter.py

```diff
@@ -223,7 +223,7 @@ def make_options():
                     "type": "regexp_csv",
                     "metavar": "<pattern>[,<pattern>...]",
                     "dest": "black_list_re",
-                    "default": (),
+                    "default": (r"^\.#",),
                     "help": "Files or directories matching the regex patterns are"
                     " skipped. The regex matches against base names, not paths.",
```

Comment:
Let's add something about the new default value here. Something along the lines of: the default value ignores emacs file locks.

Reply:
```suggestion
                    " skipped. The regex matches against base names, not paths. The default value "
                    "ignores emacs file locks",
```